start-local-agency.ps1

```
@@ -0,0 +1,39 @@
+$py = if (Test-Path "$ROOT\.venv\Scripts\python.exe") { "$ROOT\.venv\Scripts\python.exe" } else { "python" }
+
+# 3) Start the agency backend (.env forces the local brain; RUN_BACKGROUND_IN_WEB=true runs CEO/dispatch/self-heal loops)
+Write-Host "[2/3] Starting agency backend on http://localhost:8001 ..." -ForegroundColor Cyan
+Start-Process $py -ArgumentList "-m","uvicorn","backend.server:app","--host","127.0.0.1","--port","8001" -WorkingDirectory $ROOT -WindowStyle Minimized
+$up=$false
+for ($i=0; $i -lt 90; $i++) { Start-Sleep 1; try { Invoke-RestMethod "http://localhost:8001/api/doctor/public" -TimeoutSec 2 | Out-Null; $up=$true; break } catch {} }
+if ($up) { Write-Host "[OK] Backend healthy" -ForegroundColor Green } else { Write-Host "Backend slow to start - check the minimized Python window." -ForegroundColor Yellow }
+
+# 4) Kick the autonomy cycle (CEO + dispatch + self-healing + GitHub issue/PR pickup)
+Write-Host "[3/3] Triggering autonomy cycle..." -ForegroundColor Cyan
+try { Invoke-RestMethod "http://localhost:8001/api/autonomy/status" -TimeoutSec 60 | Out-Null } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)